src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -107,6 +102,26 @@ def try_run_vipin(log_path: Path) -> None:
         logger.warning(f"Performance reporting failed with: {e}")
 
 
+def run_rq_dashboard(redis_urls: list, output_directory: Path) -> None:
```

Comment:
I'd make a `monitoring` module and put this function in it. It'll eventually become a subpackage with a bunch of other stuff in it.

Reply:
Also, call your `output_directory` something more descriptive like `logging_root`.

Reply:
Ok I will do this.